packages/clay-card/src/ClayHorizontalCard.js

```diff
@@ -0,0 +1,138 @@
+import 'clay-checkbox';
+import 'clay-dropdown';
+import 'clay-link';
+import 'clay-sticker';
+import Component from 'metal-component';
+import defineWebComponent from 'metal-web-component';
+import Soy from 'metal-soy';
+import {Config} from 'metal-state';
+
+import actionItemsValidator from './action_items_validator';
+import templates from './ClayHorizontalCard.soy.js';
+
+/**
+ * Metal ClayHorizontalCard component.
+ */
+class ClayHorizontalCard extends Component {}
```

Comment:
It would not be better `ClayFolderCard` since this component is intended for this use case, could generate some confusion. What do you think?